### PR TITLE
Remove maintenance.html

### PR DESCRIPTION
**Pull request by maestromac**
## What type of PR is this? (check all applicable)
- [x] Feature?

## Description
~I am submitting a PR on Ben's behalf~

We decided that since this file is situated in a different location, it doesn't make sense to have this in the repo.
## Related Tickets & Documents
kind of related https://github.com/forem/internalEngineering/issues/405
## Added tests?
- [x] no, because they aren't needed

## Added to documentation?
- [x] no documentation needed

## [optional] Are there any post deployment tasks we need to perform?
n/a

### CODE REVIEW

File: public/maintenance.html [hunk 1]
```diff
@@ -1,39 +1,55 @@
-
 <!DOCTYPE html>
-<html>
-<head>
-  <title>We are undergoing maintenance</title>
+<!-- saved from url=(0057)https://s3.amazonaws.com/thepracticaldev/maintenance.html -->
+<html><head><meta http-equiv="Content-Type" content="text/html; charset=windows-1252">
```
Comment (djuber) [inline public/maintenance.html:3]: `charset=windows-1252` is this needed? (I honestly don't know, but not using utf-8 sounds interesting)

**Review by jgaskins — COMMENTED** [review]
Does updating this file automatically update the maintenance page on S3 or do we need to do that manually?

**Comment by maestromac** [comment]
@jgaskins thanks for pointing that out. yes it needs to be on s3

File: public/maintenance.html (hunk 1 above)
Comment (benhalpern) [reply public/maintenance.html:3]: I actually think this is just some copypasta — no specific intentions here.

Comment (maestromac) [reply public/maintenance.html]: Updated!

**Comment by benhalpern** [comment]
Does it actually make sense for this to even live in the repo at all? It has to be stored elsewhere because the whole point is that the site is down, and this particular one is fairly DEV-centric.

Maybe we store this in a separate utility repo for things like this?

**Comment by jgaskins** [comment]
@benhalpern I was thinking the same thing. Anything you need to show during planned (or even unplanned) downtime is specific to that deployment. Ends up as infrastructure rather than application.

**Comment by jgaskins** [comment]
Either way I absolutely love this new maintenance page

**Review by djuber — APPROVED** [review]
All red PR? :+1:

**Review by juliannatetreault — APPROVED** [review]

**Pull request merged**